ui/app/styles/helper-classes/colors.scss

```diff
@@ -81,3 +84,7 @@ select.has-error-border,
 .has-text-danger {
   color: $red-500 !important;
 }
+
+.has-text-primary {
+  color: $blue;
```

Comment:
Lame, I know, but we have one for this already. It's poorly named, but I couldn't rename it without changing various instances of it. `has-text-info` in the same file.

Reply:
blue-500 is also blue. Again LAME. But it is the way of the old patterns that we can't shake quite yet.

Reply:
Good eye! What do you think about changing the name of `has-text-info` to `has-text-primary` to better align with `is-primary`? There are only 5 instances of that class being used so it would be an easy change.

Reply:
I'm fine leaving it too if folks are familiar with that class and changing it would lead to confusion.

Reply:
Change away! If it reads better, I'm all about cleaning up the naming as we go.

Reply:
I opted to leave as is for now and remove the `has-text-primary` class. Info aligns with success and danger so I think it's better not to break that pattern up.